tests/test_optimize.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+from unittest import TestCase
+
+from aspire.utils import ensure
+from aspire.utils.matlab_compat import m_reshape
+from aspire.utils.optimize import conj_grad
+
+
+def identity(x):
```

Comment:
Why is this needed? It's not used.

Reply:
Removed in new version.

Reply:
The `preconditioner` needs to be initialized for this customized CG method. the `precond_fun` was removed and the `identity` was restored in the commit 6f43523.

Reply:
> The preconditioner needs to be initialized for this customized CG method.

That's a bug. Why isn't it provided as a default in `conj_grad`? Let's make an issue.

Reply:
The Matlab version also needs  `identity` as default preconditioner. New issue #249 was created.

Reply:
The issue #249 was closed since we are using default `identity` preconditioner.